### DISCUSSION TITLE
Can you login? I can't...

### DISCUSSION THREAD

**Discussion by wilsolutions** [codemagic-ci-cd/codemagic-docs#2721] [Q&A] [answered]
### Use case

```Text
CodeMagic is suddenly failing to login, I can't login, it display the following error:
Verification failed
Something went wrong. Please try again.
```

### Idea

```Text
Please fix the login
```

**Comment by involvepr** [comment]
I can't log in either. It seems like a bug.
I opened a ticket with Codemagic support, they are reviewing it.

**Comment by icarusdust** [comment, marked as answer]
@wilsolutions @involvepr We have identified the cause of the issue and managed to resolve it. We will continue to monitor the situation to make sure everything is fixed. Sorry for the inconvenience.

**Reply by wilsolutions** [reply]
Thank you!
